core/relay/docker/testnet-envs/.env.corda

```diff
@@ -6,6 +6,8 @@ DRIVER_PORT=9099
 DRIVER_NAME=corda-driver-Corda_Network
 RELAY_NAME=relay-corda
 RELAY_PORT=9081
+DB_OPEN_MAX_RETRIES=100
+DB_OPEN_RETRY_BACKOFF_TIME=100
```

Comment:
Again, this should be 250 and 20 respectively, no?

Reply:
Changed to 500, 10 for relay.